### PR TITLE
[lib, numlib]{SYSTEM} cuDNN-9.5.0.5, cuTENSOR-2.0.2.5

### PR DESCRIPTION
**Pull request by ankekreuzer**
Some more packages from JSC

### CODE REVIEW

**Comment by ankekreuzer** [comment]
Test report by @ankekreuzer
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in total)
zam329 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i5-8250U CPU @ 1.60GHz, Python 3.9.18
See https://gist.github.com/ankekreuzer/a10f02c1b2702a3d51278cbc028ea4a6 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

File: easybuild/easyconfigs/c/cuDNN/cuDNN-9.5.0.50-CUDA-12.6.0.eb [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+# note: cuDNN is tied to specific to CUDA versions,
+# see also https://docs.nvidia.com/deeplearning/cudnn/support-matrix/index.html#cudnn-cuda-hardware-versions
+local_short_ver = '.'.join(version.split('.')[:3])
+local_cuda_major = '12'
+
+source_urls = [
+    'https://developer.download.nvidia.com/compute/cudnn/redist/cudnn/linux-%(cudnnarch)s/'
+]
+sources = ['%%(namelower)s-linux-%%(cudnnarch)s-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major]
+checksums = ['86e4e4f4c09b31d3850b402d94ea52741a2f94c2f717ddc8899a14aca96e032d']
```
Comment (SebastianAchilles) [inline easybuild/easyconfigs/c/cuDNN/cuDNN-9.5.0.50-CUDA-12.6.0.eb:19]:
```suggestion
checksums = [{
    '%%(namelower)s-linux-sbsa-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major:
        '494b640a69feb40ce806a726aa63a1de6b2ec459acbe6a116ef6fe3e6b27877d',
    '%%(namelower)s-linux-x86_64-%%(version)s_cuda%s-archive.tar.xz' % local_cuda_major:
        '86e4e4f4c09b31d3850b402d94ea52741a2f94c2f717ddc8899a14aca96e032d',
}]
```

**Review by SebastianAchilles — CHANGES_REQUESTED** [review]
We should also add the checksum for sbsa in cuDNN

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
bwd-rockylinux-94 - Linux Rocky Linux 9.4 (Blue Onyx), x86_64, Intel(R) Core(TM) i7-6900K CPU @ 3.20GHz (broadwell), 2 x NVIDIA NVIDIA GeForce GTX 1060 6GB, 535.216.01, Python 3.9.18
See https://gist.github.com/SebastianAchilles/38ba4fbb8eb6ca70799d3f4db7809273 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
zen2-rockylinux-810 - Linux Rocky Linux 8.10, x86_64, AMD EPYC 7452 32-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/SebastianAchilles/320667dfdf982388a56cc9f1792b97a0 for a full test report.

**Comment by SebastianAchilles** [comment]
Test report by @sebastianachilles
**SUCCESS**
Build succeeded for 3 out of 3 (3 easyconfigs in total)
skl-almalinux-94 - Linux AlmaLinux 9.4 (Seafoam Ocelot), x86_64, Intel(R) Core(TM) i7-8700 CPU @ 3.20GHz (skylake), Python 3.9.18
See https://gist.github.com/SebastianAchilles/75eec34cc94c094ba24b74919e5a5a23 for a full test report.

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ jsc-zen3

**Comment by SebastianAchilles** [comment]
@boegelbot please test @ generoso

**Comment by SebastianAchilles** [comment]
`tornado-6.4.1-GCCcore-13.3.0.eb` was already merged in https://github.com/easybuilders/easybuild-easyconfigs/pull/21663

**Review by SebastianAchilles — APPROVED** [review]
lgtm

**Pull request merged**